s3 persistent data restore failed

thanks for the awesome project!

```
environment:
    SERVICES: s3
    DEFAULT_REGION: us-east-1
    DEBUG: 1
    DATA_DIR: /tmp/localstack/data
volumes:
    - './var/localstack:/tmp/localstack'
```

It works fine on the first "docker-compose up".
Restart "docker-compose" for s3 persistent volume test
The restore will fail.




```

OUT > Waiting for all LocalStack services to be ready
OUT > 2019-08-20 10:24:29,263 CRIT Supervisor is running as root.  Privileges were not dropped because no user is specified in the config file.  If you intend to run as root, you can set user=root in the config file to avoid this message.
OUT > 2019-08-20 10:24:29,264 INFO supervisord started with pid 12
OUT > 2019-08-20 10:24:30,267 INFO spawned: 'dashboard' with pid 20
OUT > 2019-08-20 10:24:30,269 INFO spawned: 'infra' with pid 21
OUT > 2019-08-20 10:24:30,274 INFO success: dashboard entered RUNNING state, process has stayed up for > than 0 seconds (startsecs)
OUT > (. .venv/bin/activate; bin/localstack web)
OUT > (. .venv/bin/activate; exec bin/localstack start --host)
OUT > 2019-08-20 10:24:31,282 INFO success: infra entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
OUT > Starting local dev environment. CTRL-C to quit.
OUT > cannot import name 'dns_server'
OUT > Starting mock S3 (http port 4572)...
OUT > Waiting for all LocalStack services to be ready
OUT > 2019-08-20 10:24:39,809:API: 127.0.0.1 - - [20/Aug/2019 10:24:39] "GET / HTTP/1.1" 200 -
OUT > 2019-08-20 10:24:39,819:API: 127.0.0.1 - - [20/Aug/2019 10:24:39] "PUT /max HTTP/1.1" 200 -
OUT > 2019-08-20 10:24:39,829:API: 127.0.0.1 - - [20/Aug/2019 10:24:39] "PUT /max/a289c68170df2904d63e4e24bab97213ca8711da.jpg HTTP/1.1" 200 -
OUT > 2019-08-20 10:24:39,841:API: 127.0.0.1 - - [20/Aug/2019 10:24:39] "PUT /max/616569dd2b470e44277ad2705c6d1f6cf900747c.jpg HTTP/1.1" 200 -
OUT > 2019-08-20 10:24:39,853:API: 127.0.0.1 - - [20/Aug/2019 10:24:39] "PUT /max/59dbdef9a2fd1d33a2e5f73b16bfa4bd5bafd5f4.jpg HTTP/1.1" 200 -
OUT > 2019-08-20 10:24:39,861:API: 127.0.0.1 - - [20/Aug/2019 10:24:39] "PUT /max/ae5cd76b42c42aaa315b048f3bcc3ea4541e4945.jpg HTTP/1.1" 200 -
OUT > 2019-08-20 10:24:39,872:API: 127.0.0.1 - - [20/Aug/2019 10:24:39] "PUT /max/a0df2ea4bc6054f14f8af989dc427d2eb195b9c9.jpg HTTP/1.1" 200 -
OUT > 2019-08-20 10:24:39,941:API: 127.0.0.1 - - [20/Aug/2019 10:24:39] "PUT /max/5e3be602b7e3fd317eb07fcddd77c9238ab00eed.gif HTTP/1.1" 200 -
OUT > 2019-08-20 10:24:40,088:API: 127.0.0.1 - - [20/Aug/2019 10:24:40] "PUT /max/e548b034515fe47f4fea7f372b1a9738492146ef.gif HTTP/1.1" 200 -
OUT > 2019-08-20 10:24:40,138:API: 127.0.0.1 - - [20/Aug/2019 10:24:40] "PUT /max/97db749c2196b2027e1665a1f99e5dc796d5378f.jpg HTTP/1.1" 200 -
OUT > 2019-08-20 10:24:40,160:API: 127.0.0.1 - - [20/Aug/2019 10:24:40] "PUT /max/439f08e7795d3cc9045f6df4c62da2fa3019b854.jpg HTTP/1.1" 200 -
OUT > 2019-08-20 10:24:40,169:API: 127.0.0.1 - - [20/Aug/2019 10:24:40] "PUT /max/ea256f899199af63a0a6ed91320ad73e77513a4e.jpg HTTP/1.1" 200 -
OUT > 2019-08-20 10:24:40,177:API: 127.0.0.1 - - [20/Aug/2019 10:24:40] "PUT /max/fc26d68d849d8dc9f8fac8681aee2e17670fa800.jpg HTTP/1.1" 200 -
OUT > Error starting infrastructure: Expecting ',' delimiter: line 1 column 16387 (char 16386) Traceback (most recent call last):
OUT >   File "/opt/code/localstack/localstack/services/infra.py", line 392, in start_infra
OUT >     restore_persisted_data(apis=apis)
OUT >   File "/opt/code/localstack/localstack/services/infra.py", line 210, in restore_persisted_data
OUT >     persistence.restore_persisted_data(api)
OUT >   File "/opt/code/localstack/localstack/utils/persistence.py", line 92, in restore_persisted_data
OUT >     return replay(api)
OUT >   File "/opt/code/localstack/localstack/utils/persistence.py", line 83, in replay
OUT >     command = json.loads(line)
OUT >   File "/usr/lib/python3.6/json/__init__.py", line 354, in loads
OUT >     return _default_decoder.decode(s)
OUT >   File "/usr/lib/python3.6/json/decoder.py", line 339, in decode
OUT >     obj, end = self.raw_decode(s, idx=_w(s, 0).end())
OUT >   File "/usr/lib/python3.6/json/decoder.py", line 355, in raw_decode
OUT >     obj, end = self.scan_once(s, idx)
OUT > json.decoder.JSONDecodeError: Expecting ',' delimiter: line 1 column 16387 (char 16386)
OUT > 2019-08-20 10:24:42,574 INFO exited: infra (exit status 2; not expected)
OUT > 2019-08-20 10:24:42,576 INFO spawned: 'infra' with pid 75
OUT > Waiting for all LocalStack services to be ready
OUT > Traceback (most recent call last):
OUT >   File "bin/localstack", line 86, in <module>
OUT >     subcommand['function'](argv, args)
OUT >   File "/opt/code/localstack/localstack/utils/cli.py", line 34, in cmd_infra
OUT >     bootstrap.start_infra_locally()
OUT >   File "/opt/code/localstack/localstack/utils/bootstrap.py", line 223, in start_infra_locally
OUT >     return infra.start_infra()
OUT >   File "/opt/code/localstack/localstack/services/infra.py", line 407, in start_infra
OUT >     raise e
OUT >   File "/opt/code/localstack/localstack/services/infra.py", line 392, in start_infra
OUT >     restore_persisted_data(apis=apis)
OUT >   File "/opt/code/localstack/localstack/services/infra.py", line 210, in restore_persisted_data
OUT >     persistence.restore_persisted_data(api)
OUT >   File "/opt/code/localstack/localstack/utils/persistence.py", line 92, in restore_persisted_data
OUT >     return replay(api)
OUT >   File "/opt/code/localstack/localstack/utils/persistence.py", line 83, in replay
OUT >     command = json.loads(line)
OUT >   File "/usr/lib/python3.6/json/__init__.py", line 354, in loads
OUT >     return _default_decoder.decode(s)
OUT >   File "/usr/lib/python3.6/json/decoder.py", line 339, in decode
OUT >     obj, end = self.raw_decode(s, idx=_w(s, 0).end())
OUT >   File "/usr/lib/python3.6/json/decoder.py", line 355, in raw_decode
OUT >     obj, end = self.scan_once(s, idx)
OUT > json.decoder.JSONDecodeError: Expecting ',' delimiter: line 1 column 16387 (char 16386)
OUT > 
OUT > ERROR: Expecting ',' delimiter: line 1 column 16387 (char 16386)
OUT > (. .venv/bin/activate; exec bin/localstack start --host)
OUT > make: *** [Makefile:47: infra] Error 1
OUT > 2019-08-20 10:24:43,585 INFO success: infra entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
OUT > Starting local dev environment. CTRL-C to quit.
OUT > cannot import name 'dns_server'
OUT > Starting mock S3 (http port 4572)...
OUT > Waiting for all LocalStack services to be ready
OUT > 2019-08-20 10:24:49,607:API: 127.0.0.1 - - [20/Aug/2019 10:24:49] "GET / HTTP/1.1" 200 -
OUT > 2019-08-20 10:24:49,617:API: 127.0.0.1 - - [20/Aug/2019 10:24:49] "PUT /max HTTP/1.1" 200 -
OUT > 2019-08-20 10:24:49,626:API: 127.0.0.1 - - [20/Aug/2019 10:24:49] "PUT /max/a289c68170df2904d63e4e24bab97213ca8711da.jpg HTTP/1.1" 200 -
OUT > 2019-08-20 10:24:49,637:API: 127.0.0.1 - - [20/Aug/2019 10:24:49] "PUT /max/616569dd2b470e44277ad2705c6d1f6cf900747c.jpg HTTP/1.1" 200 -
OUT > 2019-08-20 10:24:49,648:API: 127.0.0.1 - - [20/Aug/2019 10:24:49] "PUT /max/59dbdef9a2fd1d33a2e5f73b16bfa4bd5bafd5f4.jpg HTTP/1.1" 200 -
OUT > 2019-08-20 10:24:49,656:API: 127.0.0.1 - - [20/Aug/2019 10:24:49] "PUT /max/ae5cd76b42c42aaa315b048f3bcc3ea4541e4945.jpg HTTP/1.1" 200 -
OUT > 2019-08-20 10:24:49,666:API: 127.0.0.1 - - [20/Aug/2019 10:24:49] "PUT /max/a0df2ea4bc6054f14f8af989dc427d2eb195b9c9.jpg HTTP/1.1" 200 -
OUT > 2019-08-20 10:24:49,725:API: 127.0.0.1 - - [20/Aug/2019 10:24:49] "PUT /max/5e3be602b7e3fd317eb07fcddd77c9238ab00eed.gif HTTP/1.1" 200 -
OUT > 2019-08-20 10:24:49,860:API: 127.0.0.1 - - [20/Aug/2019 10:24:49] "PUT /max/e548b034515fe47f4fea7f372b1a9738492146ef.gif HTTP/1.1" 200 -
OUT > 2019-08-20 10:24:49,917:API: 127.0.0.1 - - [20/Aug/2019 10:24:49] "PUT /max/97db749c2196b2027e1665a1f99e5dc796d5378f.jpg HTTP/1.1" 200 -
OUT > 2019-08-20 10:24:49,937:API: 127.0.0.1 - - [20/Aug/2019 10:24:49] "PUT /max/439f08e7795d3cc9045f6df4c62da2fa3019b854.jpg HTTP/1.1" 200 -
OUT > 2019-08-20 10:24:49,945:API: 127.0.0.1 - - [20/Aug/2019 10:24:49] "PUT /max/ea256f899199af63a0a6ed91320ad73e77513a4e.jpg HTTP/1.1" 200 -
OUT > 2019-08-20 10:24:49,954:API: 127.0.0.1 - - [20/Aug/2019 10:24:49] "PUT /max/fc26d68d849d8dc9f8fac8681aee2e17670fa800.jpg HTTP/1.1" 200 -
OUT > Error starting infrastructure: Expecting ',' delimiter: line 1 column 16387 (char 16386) Traceback (most recent call last):
OUT >   File "/opt/code/localstack/localstack/services/infra.py", line 392, in start_infra
OUT >     restore_persisted_data(apis=apis)
OUT >   File "/opt/code/localstack/localstack/services/infra.py", line 210, in restore_persisted_data
OUT >     persistence.restore_persisted_data(api)
OUT >   File "/opt/code/localstack/localstack/utils/persistence.py", line 92, in restore_persisted_data
OUT >     return replay(api)
OUT >   File "/opt/code/localstack/localstack/utils/persistence.py", line 83, in replay
OUT >     command = json.loads(line)
OUT >   File "/usr/lib/python3.6/json/__init__.py", line 354, in loads
OUT >     return _default_decoder.decode(s)
OUT >   File "/usr/lib/python3.6/json/decoder.py", line 339, in decode
OUT >     obj, end = self.raw_decode(s, idx=_w(s, 0).end())
OUT >   File "/usr/lib/python3.6/json/decoder.py", line 355, in raw_decode
OUT >     obj, end = self.scan_once(s, idx)
OUT > json.decoder.JSONDecodeError: Expecting ',' delimiter: line 1 column 16387 (char 16386)
OUT > Traceback (most recent call last):
OUT >   File "bin/localstack", line 86, in <module>
OUT >     subcommand['function'](argv, args)
OUT >   File "/opt/code/localstack/localstack/utils/cli.py", line 34, in cmd_infra
OUT >     bootstrap.start_infra_locally()
OUT >   File "/opt/code/localstack/localstack/utils/bootstrap.py", line 223, in start_infra_locally
OUT >     return infra.start_infra()
OUT >   File "/opt/code/localstack/localstack/services/infra.py", line 407, in start_infra
OUT >     raise e
OUT >   File "/opt/code/localstack/localstack/services/infra.py", line 392, in start_infra
OUT >     restore_persisted_data(apis=apis)
OUT >   File "/opt/code/localstack/localstack/services/infra.py", line 210, in restore_persisted_data
OUT >     persistence.restore_persisted_data(api)
OUT >   File "/opt/code/localstack/localstack/utils/persistence.py", line 92, in restore_persisted_data
OUT >     return replay(api)
OUT >   File "/opt/code/localstack/localstack/utils/persistence.py", line 83, in replay
OUT >     command = json.loads(line)
OUT >   File "/usr/lib/python3.6/json/__init__.py", line 354, in loads
OUT >     return _default_decoder.decode(s)
OUT >   File "/usr/lib/python3.6/json/decoder.py", line 339, in decode
OUT >     obj, end = self.raw_decode(s, idx=_w(s, 0).end())
OUT >   File "/usr/lib/python3.6/json/decoder.py", line 355, in raw_decode
...
```

Endlessly repeating

```
...
  File "/opt/code/localstack/localstack/utils/persistence.py", line 83, in replay
OUT >     command = json.loads(line)
OUT >   File "/usr/lib/python3.6/json/__init__.py", line 354, in loads
OUT >     return _default_decoder.decode(s)
OUT >   File "/usr/lib/python3.6/json/decoder.py", line 339, in decode
OUT >     obj, end = self.raw_decode(s, idx=_w(s, 0).end())
OUT >   File "/usr/lib/python3.6/json/decoder.py", line 355, in raw_decode
OUT >     obj, end = self.scan_once(s, idx)
OUT > json.decoder.JSONDecodeError: Expecting ',' delimiter: line 1 column 16387 (char 16386)
OUT > 
OUT > ERROR: Expecting ',' delimiter: line 1 column 16387 (char 16386)
OUT > (. .venv/bin/activate; exec bin/localstack start --host)
OUT > make: *** [Makefile:47: infra] Error 1
OUT > Waiting for all LocalStack services to be ready
OUT > 2019-08-20 10:33:00,507 INFO success: infra entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
OUT > Starting local dev environment. CTRL-C to quit.
OUT > cannot import name 'dns_server'
OUT > Starting mock S3 (http port 4572)...
OUT > 2019-08-20 10:33:06,498:API: 127.0.0.1 - - [20/Aug/2019 10:33:06] "GET / HTTP/1.1" 200 -
OUT > 2019-08-20 10:33:06,507:API: 127.0.0.1 - - [20/Aug/2019 10:33:06] "PUT /max HTTP/1.1" 200 -
OUT > 2019-08-20 10:33:06,516:API: 127.0.0.1 - - [20/Aug/2019 10:33:06] "PUT /max/a289c68170df2904d63e4e24bab97213ca8711da.jpg HTTP/1.1" 200 -
OUT > 2019-08-20 10:33:06,528:API: 127.0.0.1 - - [20/Aug/2019 10:33:06] "PUT /max/616569dd2b470e44277ad2705c6d1f6cf900747c.jpg HTTP/1.1" 200 -
OUT > 2019-08-20 10:33:06,541:API: 127.0.0.1 - - [20/Aug/2019 10:33:06] "PUT /max/59dbdef9a2fd1d33a2e5f73b16bfa4bd5bafd5f4.jpg HTTP/1.1" 200 -
OUT > 2019-08-20 10:33:06,549:API: 127.0.0.1 - - [20/Aug/2019 10:33:06] "PUT /max/ae5cd76b42c42aaa315b048f3bcc3ea4541e4945.jpg HTTP/1.1" 200 -
OUT > 2019-08-20 10:33:06,561:API: 127.0.0.1 - - [20/Aug/2019 10:33:06] "PUT /max/a0df2ea4bc6054f14f8af989dc427d2eb195b9c9.jpg HTTP/1.1" 200 -
OUT > 2019-08-20 10:33:06,627:API: 127.0.0.1 - - [20/Aug/2019 10:33:06] "PUT /max/5e3be602b7e3fd317eb07fcddd77c9238ab00eed.gif HTTP/1.1" 200 -
OUT > 2019-08-20 10:33:06,764:API: 127.0.0.1 - - [20/Aug/2019 10:33:06] "PUT /max/e548b034515fe47f4fea7f372b1a9738492146ef.gif HTTP/1.1" 200 -
OUT > Waiting for all LocalStack services to be ready
OUT > 2019-08-20 10:33:06,817:API: 127.0.0.1 - - [20/Aug/2019 10:33:06] "PUT /max/97db749c2196b2027e1665a1f99e5dc796d5378f.jpg HTTP/1.1" 200 -
OUT > 2019-08-20 10:33:06,838:API: 127.0.0.1 - - [20/Aug/2019 10:33:06] "PUT /max/439f08e7795d3cc9045f6df4c62da2fa3019b854.jpg HTTP/1.1" 200 -
OUT > 2019-08-20 10:33:06,847:API: 127.0.0.1 - - [20/Aug/2019 10:33:06] "PUT /max/ea256f899199af63a0a6ed91320ad73e77513a4e.jpg HTTP/1.1" 200 -
OUT > 2019-08-20 10:33:06,856:API: 127.0.0.1 - - [20/Aug/2019 10:33:06] "PUT /max/fc26d68d849d8dc9f8fac8681aee2e17670fa800.jpg HTTP/1.1" 200 -
OUT > Error starting infrastructure: Expecting ',' delimiter: line 1 column 16387 (char 16386) Traceback (most recent call last):
OUT >   File "/opt/code/localstack/localstack/services/infra.py", line 392, in start_infra
OUT >     restore_persisted_data(apis=apis)
OUT >   File "/opt/code/localstack/localstack/services/infra.py", line 210, in restore_persisted_data
OUT >     persistence.restore_persisted_data(api)
OUT >   File "/opt/code/localstack/localstack/utils/persistence.py", line 92, in restore_persisted_data
OUT >     return replay(api)
OUT >   File "/opt/code/localstack/localstack/utils/persistence.py", line 83, in replay
OUT >     command = json.loads(line)
OUT >   File "/usr/lib/python3.6/json/__init__.py", line 354, in loads
OUT >     return _default_decoder.decode(s)
OUT >   File "/usr/lib/python3.6/json/decoder.py", line 339, in decode
OUT >     obj, end = self.raw_decode(s, idx=_w(s, 0).end())
OUT >   File "/usr/lib/python3.6/json/decoder.py", line 355, in raw_decode
OUT >     obj, end = self.scan_once(s, idx)
OUT > json.decoder.JSONDecodeError: Expecting ',' delimiter: line 1 column 16387 (char 16386)
OUT > 2019-08-20 10:33:09,222 INFO exited: infra (exit status 2; not expected)
OUT > 2019-08-20 10:33:09,223 INFO spawned: 'infra' with pid 2321
OUT > Traceback (most recent call last):
OUT >   File "bin/localstack", line 86, in <module>
OUT >     subcommand['function'](argv, args)
OUT >   File "/opt/code/localstack/localstack/utils/cli.py", line 34, in cmd_infra
OUT >     bootstrap.start_infra_locally()
OUT >   File "/opt/code/localstack/localstack/utils/bootstrap.py", line 223, in start_infra_locally
OUT >     return infra.start_infra()
OUT >   File "/opt/code/localstack/localstack/services/infra.py", line 407, in start_infra
OUT >     raise e
OUT >   File "/opt/code/localstack/localstack/services/infra.py", line 392, in start_infra
OUT >     restore_persisted_data(apis=apis)
OUT >   File "/opt/code/localstack/localstack/services/infra.py", line 210, in restore_persisted_data
OUT >     persistence.restore_persisted_data(api)
OUT >   File "/opt/code/localstack/localstack/utils/persistence.py", line 92, in restore_persisted_data
OUT >     return replay(api)
OUT >   File "/opt/code/localstack/localstack/utils/persistence.py", line 83, in replay
OUT >     command = json.loads(line)
OUT >   File "/usr/lib/python3.6/json/__init__.py", line 354, in loads
OUT >     return _default_decoder.decode(s)
OUT >   File "/usr/lib/python3.6/json/decoder.py", line 339, in decode
OUT >     obj, end = self.raw_decode(s, idx=_w(s, 0).end())
OUT >   File "/usr/lib/python3.6/json/decoder.py", line 355, in raw_decode
OUT >     obj, end = self.scan_once(s, idx)
OUT > json.decoder.JSONDecodeError: Expecting ',' delimiter: line 1 column 16387 (char 16386)
OUT > 
OUT > ERROR: Expecting ',' delimiter: line 1 column 16387 (char 16386)
OUT > (. .venv/bin/activate; exec bin/localstack start --host)
OUT > make: *** [Makefile:47: infra] Error 1
OUT > 2019-08-20 10:33:10,229 INFO success: infra entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
OUT > Starting local dev environment. CTRL-C to quit.
OUT > cannot import name 'dns_server'
OUT > Starting mock S3 (http port 4572)...
OUT > Waiting for all LocalStack services to be ready
OUT > 2019-08-20 10:33:17,210:API: 127.0.0.1 - - [20/Aug/2019 10:33:17] "GET / HTTP/1.1" 200 -
OUT > 2019-08-20 10:33:17,219:API: 127.0.0.1 - - [20/Aug/2019 10:33:17] "PUT /max HTTP/1.1" 200 -
OUT > 2019-08-20 10:33:17,228:API: 127.0.0.1 - - [20/Aug/2019 10:33:17] "PUT /max/a289c68170df2904d63e4e24bab97213ca8711da.jpg HTTP/1.1" 200 -
OUT > 2019-08-20 10:33:17,239:API: 127.0.0.1 - - [20/Aug/2019 10:33:17] "PUT /max/616569dd2b470e44277ad2705c6d1f6cf900747c.jpg HTTP/1.1" 200 -
OUT > 2019-08-20 10:33:17,251:API: 127.0.0.1 - - [20/Aug/2019 10:33:17] "PUT /max/59dbdef9a2fd1d33a2e5f73b16bfa4bd5bafd5f4.jpg HTTP/1.1" 200 -
OUT > 2019-08-20 10:33:17,258:API: 127.0.0.1 - - [20/Aug/2019 10:33:17] "PUT /max/ae5cd76b42c42aaa315b048f3bcc3ea4541e4945.jpg HTTP/1.1" 200 -
OUT > 2019-08-20 10:33:17,268:API: 127.0.0.1 - - [20/Aug/2019 10:33:17] "PUT /max/a0df2ea4bc6054f14f8af989dc427d2eb195b9c9.jpg HTTP/1.1" 200 -
OUT > 2019-08-20 10:33:17,328:API: 127.0.0.1 - - [20/Aug/2019 10:33:17] "PUT /max/5e3be602b7e3fd317eb07fcddd77c9238ab00eed.gif HTTP/1.1" 200 -
OUT > 2019-08-20 10:33:17,469:API: 127.0.0.1 - - [20/Aug/2019 10:33:17] "PUT /max/e548b034515fe47f4fea7f372b1a9738492146ef.gif HTTP/1.1" 200 -
OUT > 2019-08-20 10:33:17,528:API: 127.0.0.1 - - [20/Aug/2019 10:33:17] "PUT /max/97db749c2196b2027e1665a1f99e5dc796d5378f.jpg HTTP/1.1" 200 -
OUT > 2019-08-20 10:33:17,548:API: 127.0.0.1 - - [20/Aug/2019 10:33:17] "PUT /max/439f08e7795d3cc9045f6df4c62da2fa3019b854.jpg HTTP/1.1" 200 -
OUT > 2019-08-20 10:33:17,557:API: 127.0.0.1 - - [20/Aug/2019 10:33:17] "PUT /max/ea256f899199af63a0a6ed91320ad73e77513a4e.jpg HTTP/1.1" 200 -
OUT > 2019-08-20 10:33:17,566:API: 127.0.0.1 - - [20/Aug/2019 10:33:17] "PUT /max/fc26d68d849d8dc9f8fac8681aee2e17670fa800.jpg HTTP/1.1" 200 -
OUT > Error starting infrastructure: Expecting ',' delimiter: line 1 column 16387 (char 16386) Traceback (most recent call last):
OUT >   File "/opt/code/localstack/localstack/services/infra.py", line 392, in start_infra
OUT >     restore_persisted_data(apis=apis)
OUT >   File "/opt/code/localstack/localstack/services/infra.py", line 210, in restore_persisted_data
OUT >     persistence.restore_persisted_data(api)
OUT >   File "/opt/code/localstack/localstack/utils/persistence.py", line 92, in restore_persisted_data
OUT >     return replay(api)
OUT >   File "/opt/code/localstack/localstack/utils/persistence.py", line 83, in replay
OUT >     command = json.loads(line)
OUT >   File "/usr/lib/python3.6/json/__init__.py", line 354, in loads
OUT >     return _default_decoder.decode(s)
OUT >   File "/usr/lib/python3.6/json/decoder.py", line 339, in decode
OUT >     obj, end = self.raw_decode(s, idx=_w(s, 0).end())
OUT >   File "/usr/lib/python3.6/json/decoder.py", line 355, in raw_decode
OUT >     obj, end = self.scan_once(s, idx)
OUT > json.decoder.JSONDecodeError: Expecting ',' delimiter: line 1 column 16387 (char 16386)
OUT > 
OUT > Traceback (most recent call last):
OUT >   File "bin/localstack", line 86, in <module>
OUT >     subcommand['function'](argv, args)
OUT >   File "/opt/code/localstack/localstack/utils/cli.py", line 34, in cmd_infra
OUT >     bootstrap.start_infra_locally()
OUT >   File "/opt/code/localstack/localstack/utils/bootstrap.py", line 223, in start_infra_locally
OUT >     return infra.start_infra()
OUT >   File "/opt/code/localstack/localstack/services/infra.py", line 407, in start_infra
OUT >     raise e
OUT >   File "/opt/code/localstack/localstack/services/infra.py", line 392, in start_infra
OUT >     restore_persisted_data(apis=apis)
OUT >   File "/opt/code/localstack/localstack/services/infra.py", line 210, in restore_persisted_data
OUT >     persistence.restore_persisted_data(api)
OUT >   File "/opt/code/localstack/localstack/utils/persistence.py", line 92, in restore_persisted_data
OUT >     return replay(api)
OUT >   File "/opt/code/localstack/localstack/utils/persistence.py", line 83, in replay
OUT >     command = json.loads(line)
OUT >   File "/usr/lib/python3.6/json/__init__.py", line 354, in loads
OUT >     return _default_decoder.decode(s)
OUT >   File "/usr/lib/python3.6/json/decoder.py", line 339, in decode
OUT >     obj, end = self.raw_decode(s, idx=_w(s, 0).end())
OUT >   File "/usr/lib/python3.6/json/decoder.py", line 355, in raw_decode
OUT >     obj, end = self.scan_once(s, idx)
OUT > json.decoder.JSONDecodeError: Expecting ',' delimiter: line 1 column 16387 (char 16386)
OUT > 
OUT > ERROR: Expecting ',' delimiter: line 1 column 16387 (char 16386)
OUT > 2019-08-20 10:33:19,947 INFO exited: infra (exit status 2; not expected)
OUT > 2019-08-20 10:33:19,948 INFO spawned: 'infra' with pid 2365
OUT > (. .venv/bin/activate; exec bin/localstack start --host)
OUT > make: *** [Makefile:47: infra] Error 1
OUT > 2019-08-20 10:33:20,955 INFO success: infra entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
OUT > Waiting for all LocalStack services to be ready
OUT > Starting local dev environment. CTRL-C to quit.
OUT > cannot import name 'dns_server'
OUT > Starting mock S3 (http port 4572)...
OUT > 2019-08-20 10:33:26,928:API: 127.0.0.1 - - [20/Aug/2019 10:33:26] "GET / HTTP/1.1" 200 -
OUT > 2019-08-20 10:33:26,938:API: 127.0.0.1 - - [20/Aug/2019 10:33:26] "PUT /max HTTP/1.1" 200 -
OUT > 2019-08-20 10:33:26,947:API: 127.0.0.1 - - [20/Aug/2019 10:33:26] "PUT /max/a289c68170df2904d63e4e24bab97213ca8711da.jpg HTTP/1.1" 200 -
OUT > 2019-08-20 10:33:26,957:API: 127.0.0.1 - - [20/Aug/2019 10:33:26] "PUT /max/616569dd2b470e44277ad2705c6d1f6cf900747c.jpg HTTP/1.1" 200 -
OUT > 2019-08-20 10:33:26,968:API: 127.0.0.1 - - [20/Aug/2019 10:33:26] "PUT /max/59dbdef9a2fd1d33a2e5f73b16bfa4bd5bafd5f4.jpg HTTP/1.1" 200 -
OUT > 2019-08-20 10:33:26,976:API: 127.0.0.1 - - [20/Aug/2019 10:33:26] "PUT /max/ae5cd76b42c42aaa315b048f3bcc3ea4541e4945.jpg HTTP/1.1" 200 -
OUT > 2019-08-20 10:33:26,987:API: 127.0.0.1 - - [20/Aug/2019 10:33:26] "PUT /max/a0df2ea4bc6054f14f8af989dc427d2eb195b9c9.jpg HTTP/1.1" 200 -
OUT > 2019-08-20 10:33:27,053:API: 127.0.0.1 - - [20/Aug/2019 10:33:27] "PUT /max/5e3be602b7e3fd317eb07fcddd77c9238ab00eed.gif HTTP/1.1" 200 -
OUT > 2019-08-20 10:33:27,189:API: 127.0.0.1 - - [20/Aug/2019 10:33:27] "PUT /max/e548b034515fe47f4fea7f372b1a9738492146ef.gif HTTP/1.1" 200 -
OUT > 2019-08-20 10:33:27,247:API: 127.0.0.1 - - [20/Aug/2019 10:33:27] "PUT /max/97db749c2196b2027e1665a1f99e5dc796d5378f.jpg HTTP/1.1" 200 -
OUT > 2019-08-20 10:33:27,267:API: 127.0.0.1 - - [20/Aug/2019 10:33:27] "PUT /max/439f08e7795d3cc9045f6df4c62da2fa3019b854.jpg HTTP/1.1" 200 -
OUT > 2019-08-20 10:33:27,275:API: 127.0.0.1 - - [20/Aug/2019 10:33:27] "PUT /max/ea256f899199af63a0a6ed91320ad73e77513a4e.jpg HTTP/1.1" 200 -
OUT > 2019-08-20 10:33:27,284:API: 127.0.0.1 - - [20/Aug/2019 10:33:27] "PUT /max/fc26d68d849d8dc9f8fac8681aee2e17670fa800.jpg HTTP/1.1" 200 -
OUT > Error starting infrastructure: Expecting ',' delimiter: line 1 column 16387 (char 16386) Traceback (most recent call last):
OUT >   File "/opt/code/localstack/localstack/services/infra.py", line 392, in start_infra
OUT >     restore_persisted_data(apis=apis)
OUT >   File "/opt/code/localstack/localstack/services/infra.py", line 210, in restore_persisted_data
OUT >     persistence.restore_persisted_data(api)
OUT >   File "/opt/code/localstack/localstack/utils/persistence.py", line 92, in restore_persisted_data
OUT >     return replay(api)
OUT >   File "/opt/code/localstack/localstack/utils/persistence.py", line 83, in replay
OUT >     command = json.loads(line)
OUT >   File "/usr/lib/python3.6/json/__init__.py", line 354, in loads
OUT >     return _default_decoder.decode(s)
OUT >   File "/usr/lib/python3.6/json/decoder.py", line 339, in decode
OUT >     obj, end = self.raw_decode(s, idx=_w(s, 0).end())
OUT >   File "/usr/lib/python3.6/json/decoder.py", line 355, in raw_decode
OUT >     obj, end = self.scan_once(s, idx)
OUT > json.decoder.JSONDecodeError: Expecting ',' delimiter: line 1 column 16387 (char 16386)
OUT > Waiting for all LocalStack services to be ready
OUT > 2019-08-20 10:33:29,634 INFO exited: infra (exit status 2; not expected)
OUT > 2019-08-20 10:33:29,637 INFO spawned: 'infra' with pid 2411
OUT > Traceback (most recent call last):
OUT >   File "bin/localstack", line 86, in <module>
OUT >     subcommand['function'](argv, args)
OUT >   File "/opt/code/localstack/localstack/utils/cli.py", line 34, in cmd_infra
OUT >     bootstrap.start_infra_locally()
OUT >   File "/opt/code/localstack/localstack/utils/bootstrap.py", line 223, in start_infra_locally
OUT >     return infra.start_infra()
OUT >   File "/opt/code/localstack/localstack/services/infra.py", line 407, in start_infra
OUT >     raise e
OUT >   File "/opt/code/localstack/localstack/services/infra.py", line 392, in start_infra
OUT >     restore_persisted_data(apis=apis)
OUT >   File "/opt/code/localstack/localstack/services/infra.py", line 210, in restore_persisted_data
OUT >     persistence.restore_persisted_data(api)
OUT >   File "/opt/code/localstack/localstack/utils/persistence.py", line 92, in restore_persisted_data
OUT >     return replay(api)
OUT >   File "/opt/code/localstack/localstack/utils/persistence.py", line 83, in replay
OUT >     command = json.loads(line)
OUT >   File "/usr/lib/python3.6/json/__init__.py", line 354, in loads
OUT >     return _default_decoder.decode(s)
OUT >   File "/usr/lib/python3.6/json/decoder.py", line 339, in decode
OUT >     obj, end = self.raw_decode(s, idx=_w(s, 0).end())
OUT >   File "/usr/lib/python3.6/json/decoder.py", line 355, in raw_decode
OUT >     obj, end = self.scan_once(s, idx)
OUT > json.decoder.JSONDecodeError: Expecting ',' delimiter: line 1 column 16387 (char 16386)
OUT > ERROR: Expecting ',' delimiter: line 1 column 16387 (char 16386)
OUT > (. .venv/bin/activate; exec bin/localstack start --host)
OUT > make: *** [Makefile:47: infra] Error 1
OUT > 2019-08-20 10:33:30,644 INFO success: infra entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
OUT > Starting local dev environment. CTRL-C to quit.
OUT > cannot import name 'dns_server'
OUT > Starting mock S3 (http port 4572)...
OUT > Waiting for all LocalStack services to be ready
OUT > 2019-08-20 10:33:36,629:API: 127.0.0.1 - - [20/Aug/2019 10:33:36] "GET / HTTP/1.1" 200 -
OUT > 2019-08-20 10:33:36,639:API: 127.0.0.1 - - [20/Aug/2019 10:33:36] "PUT /max HTTP/1.1" 200 -
OUT > 2019-08-20 10:33:36,648:API: 127.0.0.1 - - [20/Aug/2019 10:33:36] "PUT /max/a289c68170df2904d63e4e24bab97213ca8711da.jpg HTTP/1.1" 200 -
OUT > 2019-08-20 10:33:36,658:API: 127.0.0.1 - - [20/Aug/2019 10:33:36] "PUT /max/616569dd2b470e44277ad2705c6d1f6cf900747c.jpg HTTP/1.1" 200 -
OUT > 2019-08-20 10:33:36,681:API: 127.0.0.1 - - [20/Aug/2019 10:33:36] "PUT /max/59dbdef9a2fd1d33a2e5f73b16bfa4bd5bafd5f4.jpg HTTP/1.1" 200 -
OUT > 2019-08-20 10:33:36,688:API: 127.0.0.1 - - [20/Aug/2019 10:33:36] "PUT /max/ae5cd76b42c42aaa315b048f3bcc3ea4541e4945.jpg HTTP/1.1" 200 -
OUT > 2019-08-20 10:33:36,700:API: 127.0.0.1 - - [20/Aug/2019 10:33:36] "PUT /max/a0df2ea4bc6054f14f8af989dc427d2eb195b9c9.jpg HTTP/1.1" 200 -
OUT > 2019-08-20 10:33:36,762:API: 127.0.0.1 - - [20/Aug/2019 10:33:36] "PUT /max/5e3be602b7e3fd317eb07fcddd77c9238ab00eed.gif HTTP/1.1" 200 -
OUT > 2019-08-20 10:33:36,900:API: 127.0.0.1 - - [20/Aug/2019 10:33:36] "PUT /max/e548b034515fe47f4fea7f372b1a9738492146ef.gif HTTP/1.1" 200 -
OUT > 2019-08-20 10:33:36,957:API: 127.0.0.1 - - [20/Aug/2019 10:33:36] "PUT /max/97db749c2196b2027e1665a1f99e5dc796d5378f.jpg HTTP/1.1" 200 -
OUT > 2019-08-20 10:33:36,977:API: 127.0.0.1 - - [20/Aug/2019 10:33:36] "PUT /max/439f08e7795d3cc9045f6df4c62da2fa3019b854.jpg HTTP/1.1" 200 -
OUT > 2019-08-20 10:33:36,986:API: 127.0.0.1 - - [20/Aug/2019 10:33:36] "PUT /max/ea256f899199af63a0a6ed91320ad73e77513a4e.jpg HTTP/1.1" 200 -
OUT > 2019-08-20 10:33:36,994:API: 127.0.0.1 - - [20/Aug/2019 10:33:36] "PUT /max/fc26d68d849d8dc9f8fac8681aee2e17670fa800.jpg HTTP/1.1" 200 -
OUT > Error starting infrastructure: Expecting ',' delimiter: line 1 column 16387 (char 16386) Traceback (most recent call last):
OUT >   File "/opt/code/localstack/localstack/services/infra.py", line 392, in start_infra
OUT >     restore_persisted_data(apis=apis)
OUT >   File "/opt/code/localstack/localstack/services/infra.py", line 210, in restore_persisted_data
OUT >     persistence.restore_persisted_data(api)
OUT >   File "/opt/code/localstack/localstack/utils/persistence.py", line 92, in restore_persisted_data
OUT >     return replay(api)
OUT >   File "/opt/code/localstack/localstack/utils/persistence.py", line 83, in replay
OUT >     command = json.loads(line)
OUT >   File "/usr/lib/python3.6/json/__init__.py", line 354, in loads
OUT >     return _default_decoder.decode(s)
OUT >   File "/usr/lib/python3.6/json/decoder.py", line 339, in decode
OUT >     obj, end = self.raw_decode(s, idx=_w(s, 0).end())
OUT >   File "/usr/lib/python3.6/json/decoder.py", line 355, in raw_decode
OUT >     obj, end = self.scan_once(s, idx)
OUT > json.decoder.JSONDecodeError: Expecting ',' delimiter: line 1 column 16387 (char 16386)
OUT > 
OUT > 2019-08-20 10:33:39,343 INFO exited: infra (exit status 2; not expected)
OUT > 2019-08-20 10:33:39,346 INFO spawned: 'infra' with pid 2455
OUT > Traceback (most recent call last):
OUT >   File "bin/localstack", line 86, in <module>
OUT >     subcommand['function'](argv, args)
OUT >   File "/opt/code/localstack/localstack/utils/cli.py", line 34, in cmd_infra
OUT >     bootstrap.start_infra_locally()
OUT >   File "/opt/code/localstack/localstack/utils/bootstrap.py", line 223, in start_infra_locally
OUT >     return infra.start_infra()
OUT >   File "/opt/code/localstack/localstack/services/infra.py", line 407, in start_infra
OUT >     raise e
OUT >   File "/opt/code/localstack/localstack/services/infra.py", line 392, in start_infra
OUT >     restore_persisted_data(apis=apis)
OUT >   File "/opt/code/localstack/localstack/services/infra.py", line 210, in restore_persisted_data
OUT >     persistence.restore_persisted_data(api)
OUT >   File "/opt/code/localstack/localstack/utils/persistence.py", line 92, in restore_persisted_data
OUT >     return replay(api)
OUT >   File "/opt/code/localstack/localstack/utils/persistence.py", line 83, in replay
OUT >     command = json.loads(line)
OUT >   File "/usr/lib/python3.6/json/__init__.py", line 354, in loads
OUT >     return _default_decoder.decode(s)
OUT >   File "/usr/lib/python3.6/json/decoder.py", line 339, in decode
OUT >     obj, end = self.raw_decode(s, idx=_w(s, 0).end())
OUT >   File "/usr/lib/python3.6/json/decoder.py", line 355, in raw_decode
OUT >     obj, end = self.scan_once(s, idx)
OUT > json.decoder.JSONDecodeError: Expecting ',' delimiter: line 1 column 16387 (char 16386)
OUT > 
OUT > ERROR: Expecting ',' delimiter: line 1 column 16387 (char 16386)
OUT > (. .venv/bin/activate; exec bin/localstack start --host)
OUT > make: *** [Makefile:47: infra] Error 1
OUT > 2019-08-20 10:33:40,351 INFO success: infra entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
OUT > Starting local dev environment. CTRL-C to quit.
OUT > cannot import name 'dns_server'
OUT > Waiting for all LocalStack services to be ready
OUT > Starting mock S3 (http port 4572)...
OUT > 2019-08-20 10:33:47,317:API: 127.0.0.1 - - [20/Aug/2019 10:33:47] "GET / HTTP/1.1" 200 -
OUT > 2019-08-20 10:33:47,327:API: 127.0.0.1 - - [20/Aug/2019 10:33:47] "PUT /max HTTP/1.1" 200 -
OUT > 2019-08-20 10:33:47,337:API: 127.0.0.1 - - [20/Aug/2019 10:33:47] "PUT /max/a289c68170df2904d63e4e24bab97213ca8711da.jpg HTTP/1.1" 200 -
OUT > 2019-08-20 10:33:47,350:API: 127.0.0.1 - - [20/Aug/2019 10:33:47] "PUT /max/616569dd2b470e44277ad2705c6d1f6cf900747c.jpg HTTP/1.1" 200 -...

```

--- a/bin/docker-entrypoint.sh
+++ b/bin/docker-entrypoint.sh
@@ -11,6 +11,9 @@ source <(
   sed -ne 's/^LOCALSTACK_\([^=]\+\)=.*/export \1=${LOCALSTACK_\1}/p'
 )
 
+cat /dev/null > /tmp/localstack_infra.log
+cat /dev/null > /tmp/localstack_infra.err
+
 supervisord -c /etc/supervisord.conf &
 
 function run_startup_scripts {
@@ -30,6 +33,4 @@ function run_startup_scripts {
 
 run_startup_scripts &
 
-touch /tmp/localstack_infra.log
-touch /tmp/localstack_infra.err
 tail -qF /tmp/localstack_infra.log /tmp/localstack_infra.err

